allennlp/modules/encoder_base.py

```diff
@@ -105,8 +105,8 @@ def sort_and_run_forward(self,
             if hidden_state is None:
                 initial_states = hidden_state
             elif isinstance(hidden_state, tuple):
-                initial_states = (state.index_select(1, sorting_indices)[:, :num_valid, :]
-                                  for state in hidden_state)
+                initial_states = list(state.index_select(1, sorting_indices)[:, :num_valid, :]
```

Comment:
list comprehension would be more idiomatic

Reply:
Thanks, you're right, that was checking whether I found the issue, and I didn't look at it again after the test passed.